devimint/src/devfed.rs

```diff
@@ -52,6 +52,10 @@ pub async fn dev_fed(process_mgr: &ProcessManager) -> Result<DevFed> {
 
     info!(target: LOG_DEVIMINT, "federation and gateways started");
 
+    std::env::set_var("GWID_CLN", gw_cln.gateway_id().await?);
+    std::env::set_var("GWID_LND", gw_lnd.gateway_id().await?);
```

Comment:
Was going to suggest prefixing with `FM_` but then realized that the only reason we did this in the past was to be able to copy all relevant envirment variables into the mprocs shell IIRC. But devimint fixes this for us now. So it would probably make more sense to remove the `FM_` prefixes!

Reply:
It's good to have project-specfic prefixes in long living env variables, as it makes chances of accidental conflicts lower. So we should keep `LM_`. @m1sterc001guy @justinmoon